exercise3-1/src/camera_demo/src/main.cc

```diff
@@ -21,11 +21,7 @@ int main(int argc, char** argv) {
   RegisterThreadTracer(object_detector_tracer);
   RegisterThreadTracer(data_logger_tracer);
 
-  rclcpp::executors::SingleThreadedExecutor executor;
-
-  executor.add_node(camera_processing_node);
-  executor.add_node(actuation_node);
-  executor.spin();
+  // TODO: Create executor, add nodes to it, and call spin() on executor
```

Comment:
Do we even need this? Can't we just leave it and simply get people to run this example without actually getting them to write any code for this exercise? This will leave more time for exercise3-2, which is the star of the show.

Reply:
Amended this comment to include direction where they can lookup how to use an executor

Reply:
Yeah you have a good point. Let's leave it out as you have it here.

That said, I was playing with the schedule last night because of the break being 30 minutes. that means we have to cut 15-20 minutes in both morning and afternoon. One of the thing we discussed in the meeting is that we might need to cut some exercises. Exercise 3-1 and 4-1 are the candidate as they are less important. If we have to cut something to save file, what we can do is get people to either just copy the solutions into the file or just show them the trace. We'll have to improvise a little bit on the spot.